main/sumac/plugins.yml

```diff
@@ -95,42 +95,42 @@
 - name: mfe
   src: tutor-mfe>=19.0.0,<20.0.0
   url: https://github.com/overhangio/tutor-mfe
-  author: Edly <regis.behmo@edly.io>
-  maintainer: Edly <regis.behmo@edly.io>
+  author: Edly <hello@edly.io>
+  maintainer: Axim <abrandes@axim.org>
   description: |
     Easy integration of custom microfrontends with your Open edX platform.
     Microfrontends form the backbone of Open edX. Thus, this plugin is required by most Open edX installations.
 
 - name: minio
   src: tutor-minio>=19.0.0,<20.0.0
   url: https://github.com/overhangio/tutor-minio
-  author: Edly <regis.behmo@edly.io>
-  maintainer: Edly <regis.behmo@edly.io>
+  author: Edly <hello@edly.io>
+  maintainer: Edly <danyal.faheem@arbisoft.com>
   description: |
     Scalable object storage for Open edX.
     This plugins is pretty much a requirement to run Open edX on Kubernetes.
 
 - name: notes
   src: tutor-notes>=19.0.0,<20.0.0
   url: https://github.com/overhangio/tutor-notes
-  author: Edly <regis.behmo@edly.io>
-  maintainer: eduNEXT
+  author: Edly <hello@edly.io>
+  maintainer: eduNEXT <jhony.avella@edunext.co>
```

Comment:
@jfavellar90 are you OK with this? (and change below)

Reply:
@jfavellar90 Pinging for ☝🏽